lib/new_relic/agent/llm_event/chat_completion/message.rb

```diff
@@ -0,0 +1,32 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic
+  module Agent
+    class LlmEvent
+      class ChatCompletion
+        class Message < ChatCompletion
```

Comment:
I think I prefer or perhaps am simply accustomed to module namespacing instead of class namespacing. What does the `class class class` give us here?